### DISCUSSION TITLE
Approved but can not access any benefits

### DISCUSSION THREAD

**Discussion by ElvinRus** [community/community#121796] [GitHub Education]
### Hi everyone,

Hello guys, I have applied to github education pack. Github approved my request but I can not access any benefits. What is wrong ? It's more than 3 days

<img width="1557" alt="Screenshot 2024-05-01 at 10 21 32 AM" src="https://github.com/community/community/assets/81685361/25da8b2b-defd-4d9d-934d-6d5358281465">

**Comment by gurjeetsinghvirdee** [comment]
Hey @ElvinRus 

It will take a little bit of time to receive your benefits. You will get another email when your benefits are ready for you to claim.

**Comment by ElvinRus** [comment]
> Hey @ElvinRus
> 
> It will take a little bit of time to receive your benefits. You will get another email when your benefits are ready for you to claim.

3 days is not enough for verification ? it just approved instantly and they said I will get reply soon :)

**Reply by gurjeetsinghvirdee** [reply]
I understand your concern. You're definitely not alone in this process. Thousands of people apply for verification every day, so sometimes it takes a bit longer to process everything. Just hang tight, and you'll get a reply soon.
